cylc/flow/cfgspec/globalcfg.py

```diff
@@ -94,9 +94,10 @@
         The number of old run directory trees to retain at start-up.
     ''')
 
-    # suite
-    with Conf('cylc', desc='''
-        Default values for entries in flow.cylc ``[cylc]`` section.
+    with Conf('scheduler', desc='''
+        Default values for entries in :cylc:conf:`flow.cylc[scheduler]`
+        section. This should not be confused with scheduling in the
+        ``flow.cylc`` file.
     '''):
         Conf('UTC mode', VDR.V_BOOLEAN, False, desc='''
                 Default for :cylc:conf:`flow.cylc[cylc]UTC mode`.
```

Comment:
```suggestion
                Default for :cylc:conf:`flow.cylc[scheduler]UTC mode`.
```
A search for [cylc] shows a few of these remaining. e.g in `suite.py`.